### DISCUSSION TITLE
Como elaborar um currículo/portifólio sem saber exatamente qual área quer seguir?

### DISCUSSION THREAD

**Discussion by danielle-soaress** [frontendbr/forum#2423] [Discussão]
Então, comecei meus estudos na programação no início do ano e pouco tempo depois comecei a cursar Engenharia de Software. Atualmente, consigo desenvolver bem HTML, CSS e JavaScript, e comecei a estudar Python por conta da faculdade. 
Recentemente comecei a fazer meu portifólio pra conseguir um estágio na área, pois meu objetivo é aprofundar meus conhecimentos e ter alguma experiência prática, até mesmo pra descobrir o segmento que eu quero seguir, porque atualmente eu não sei muito bem. É exatamente nesse ponto que eu fico em dúvida, já que, por eu não saber o que quero (gosto de ambos: back-end e front-end), não sei o que colocar no meu currículo, principalmente quando se trata do meu objetivo profissional. Sinto que me destaco mais em front-end, mas não quero me limitar a essa parte, porque gosto muito de back-end e gostaria de me aprofundar mais em relação a isso.

Alguém pode dar um help?

**Comment by ninetails** [comment]
Oi Danielle,

Sobre currículo, algo que é mais trabalhoso mas você pode fazer é tentar direcionar tudo nele em relação a vaga pretendida. Se a vaga é de frontend, foque em dizer sobre frontend, porém cite backend como diferencial. O contrário se a vaga for de back. Exalte ambos caso a vaga seja de fullstack. Seu currículo precisa chamar atenção da pessoa (ou sistema automatizado) que está recrutando e geralmente o critério mais importante é ver se encaixa nos requisitos da vaga que foi aberta.
